### ISSUE TITLE
Search page copy pubspec entry button too low contrast in dark mode

### ISSUE THREAD

**Issue by parlough** [dart-lang/pub-dev#8309]
<img width="200" alt="Screenshot of copy button" src="https://github.com/user-attachments/assets/d3243b84-f0f0-4d6e-94e2-888df7d09d15">

Perhaps the contrast is fine in the general case, but when that package's card is hovered or has focus within it, it should be easier to visually identify.

It should likely also have a different hover interaction style of some sort.
